spec/itunes/volume_spec.rb

```diff
@@ -0,0 +1,47 @@
+# coding: utf-8
+require 'spec_helper'
+require 'itunes-client'
+
+include Itunes
+
+describe Volume do
+  let(:volume) { Itunes::Player.volume }
+  let(:level) { Fixnum }
```

Comment:
Oh alright, do you mean returning the volume value when calling **up** or **down** commands? if that's what you mean I agree with you, it is a good idea!
tell me if I got it wrong :3